generic/pgtclCmds.c

```diff
@@ -241,6 +242,46 @@ tcl_value(char *value)
 #define tcl_value(x) x
 #endif   /* TCL_ARRAYS */
 
+static Tcl_Encoding utf8encoding = NULL;
+
+/*
+ * Initialize utf8encoding
+ */
+void pgtclInitEncoding(Tcl_Interp *interp) {
+	utf8encoding = Tcl_GetEncoding(interp, "utf-8");
```

Comment:
Maybe check that this succeeded and is not NULL?

Reply:
Good point, done.